avocado/core/remote/runner.py

```diff
@@ -192,6 +196,7 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
         if not timeout:     # avoid timeout = 0
             timeout = None
         summary = set()
+        warning_message = "VERSIONS   : Local %s and remote %s are not the same."
```

Comment:
haha very true @ldoktor! I'll be sure to keep that in mind when I submit future pull requests.